DOCS.md

```diff
@@ -146,6 +146,9 @@ build.created
 build.started
 : unix timestamp for build started
 
+build.pull
+: pull request number (empty string if not a pull request)
+
```

Comment:
docs moved to https://github.com/drone/drone-plugin-index/blob/master/content/drone-plugins/drone-slack/index.md